### DISCUSSION TITLE
Connecting to remote side frequently stalls/fails without error

### DISCUSSION THREAD

**Discussion by nate235** [rustdesk/rustdesk#8061] [General]
### Bug Description

When connecting to remote site the connection frequently hangs or stalls until it is cancelled by the local end. Eventually after restarting rust desk the connection goes through immediately. This happens even when using my own hosted relay.

### How to Reproduce

Use the program as intended.

### Expected Behavior

Connection should proceed as usual (instant to only a few seconds delay) or a descriptive error message when it fails.

### Operating system(s) on local side and remote side

window 11 -> Win 10

### RustDesk Version(s) on local side and remote side

1.2.3-2 -> 1.2.3-2

### Screenshots

No screenshot. Again if absolutely required I could get a vid I guess.

**Comment by nate235** [comment]
Is there a reason these are being closed and moved out of the issue section to discussion? This seem atypical in my limited experience on github.

**Reply by DMSnz** [reply]
Do you by any chance run Eset Smart Security?
I had this and nearly ALWAYS got the circle of death trying to connect, every time cancel that connection, disable firewall in Eset, then connection works perfectly.
No logs, no errors no nothing in Eset, so finally removed it and installed Eset Antivirus only (no firewall) and problems are gone, instant connections every time now and so much faster it seems.

**Reply by xlionjuan** [reply]
They said! Issues is for only putting tasks

**Reply by nate235** [reply]
@DMSnz 
No I don't but that's a good start. I'll check the clients end to make sure it's not some security program/setting installed. Thanks. 

@xlionjuan 
"They said" is not a reason. I'm sure they have a good reason for it but typically issues section is where these get posted. Though devs moving issues around doesn't seem as atypical as I thought now I've have a quick read around.

**Comment by xlionjuan** [comment]
Try 1.2.4 both sides
